samples/csharp_dotnetcore/01.console-echo/README.md

```diff
@@ -8,7 +8,7 @@ This sample shows a simple echo bot and demonstrates the bot working as a consol
 
 ## Prerequisites
 
-- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 6.0

```

Comment:
Net 6 is not Core or Framework, just Net 6. If you see the changes I did for the generators, I left this as:
```suggestion
- [.NET SDK](https://dotnet.microsoft.com/download) version 6.0

```